Assignment2/src/asgn2/User.java

```diff
@@ -0,0 +1,17 @@
+package asgn2;
+
+public class User implements java.io.Serializable {
```

Comment:
Need to work on formatting your code base.

Reply:
yes sir. I am working on this assignment.